projects/fsastorefrontstyles/scss/components/checkout/_mini-cart.scss

```diff
@@ -31,3 +31,37 @@
     }
   }
 }
+
+.action-wrapper {
+  display: flex;
+  justify-content: space-between;
+  cx-fs-cart-coupon {
+    width: 60%;
+  }
+  cx-fs-cms-sync-pilot {
+    width: max-content;
+    padding-inline-end: 1rem;
+    h5 {
+      font-size: 0.875rem;
+    }
+    .link {
```

Comment:
Use BS `text-center` class for this

Reply:
Are you sure? We said this component should have minimum styles. `text-center` is BS class with rule !important, which is hard to override. With class `text-center` it looks like this:

![Screenshot 2021-12-22 at 13 06 09](https://user-images.githubusercontent.com/75082466/147090582-50c0cf36-43c4-4b92-b21a-7faef9c4ee0c.png)

If it is important to center the content I need to refactor the HTML structure and maybe add div wrapper for sync-pilot component with class text-center to center all content of component:

![Screenshot 2021-12-22 at 13 10 09](https://user-images.githubusercontent.com/75082466/147090837-7907a209-28e4-45ae-9567-c316620ef189.png)

Reply:
Yes, I overlooked that this is the styling used only for this usecase with coupon component. You are right we should have as little styling as possible on our main component.